test/bigtest/tests/sort-collection-test.js

```diff
@@ -139,24 +138,8 @@ describe('sortCollection', () => {
 });
 
 it('works with empty input', () => {
-  const output = sortCollection();
-  const isOutputCorrect = isEqual(output, []);
-
-  expect(isOutputCorrect).to.be.true;
-});
-
-it('works correctly with empty array', () => {
-  const output = sortCollection([]);
-  const isOutputCorrect = isEqual(output, []);
-
-  expect(isOutputCorrect).to.be.true;
-});
-
-it('works correctly with no arguments passed', () => {
-  const output = sortCollection();
-  const isOutputCorrect = isEqual(output, []);
-
-  expect(isOutputCorrect).to.be.true;
+  expect(sortCollection()).to.deep.equal([]);
+  expect(() => sortCollection()).not.to.throw();
```

Comment:
I think this `expect` is excess. Since there is a result in the above one, it is sure that there was no error.